Makefile

```diff
@@ -1,7 +1,8 @@
 GOTOOLS = \
 					github.com/mitchellh/gox \
 					github.com/tcnksm/ghr \
-					github.com/alecthomas/gometalinter
+					github.com/Masterminds/glide \
```

Comment:
do we need glide here? i think i made it that it is installed in get_vendor_deps if missing

Reply:
i think [this line](https://github.com/tendermint/tendermint/pull/902/files/5e677263ceba10c9349cb949b4800ebd8f9b26cb#diff-b67911656ef5d18c4ae36cb6741b7965R69) flubbed in the CI but we'll see in a moment